src/Microsoft.DocAsCode.Common/JsonUtility.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.DocAsCode.Common
 
     using Newtonsoft.Json;
 
+    using Microsoft.DocAsCode.Plugins;
+
     public static class JsonUtility
```

Comment:
todo

---
In reply to: [92966202](https://github.com/dotnet/docfx/pull/1012#discussion_r92966202) [](ancestors = 92966202)